lib/web/purchases_flutter_web.dart

```diff
@@ -162,14 +162,16 @@ class PurchasesFlutterPlugin {
       );
     }
 
-    final appUserId = arguments['appUserId'] as String?;
-
     final options = {
       'apiKey': apiKey,
-      'appUserId': appUserId,
       'flavor': _platformName,
       'flavorVersion': _pluginVersion,
     };
+    final appUserId = arguments['appUserId'] as String?;
+
+    if(appUserId != null && appUserId.isNotEmpty) {
+      options['appUserId'] = appUserId;
```

Comment:
Had to change this to be added like this since some changes in the underlying JS libraries made it break without this because we were passing null as the appUserId. A proper fix might be better in `purchases-js-hybrid-mappings`... but I thought to fix it here for now to avoid another release.